app/models/publication.rb

```diff
@@ -132,6 +132,10 @@ def self.open_access_statuses
   scope :oa_publication, -> { where(publication_type: oa_publication_types) }
   scope :non_oa_publication, -> { where.not(publication_type: oa_publication_types) }
 
+  scope :not_open_access, -> { distinct(:id).left_outer_joins(:open_access_locations).where(open_access_locations: { publication_id: nil }) }
```

Comment:
Given the larger context, this looks more like "unknown_access" than "not_open_access" to me, although that's not quite right either because there isn't enough info here to say for sure that the access is unknown. I think all that we can really determine from this query is that these are publications that don't yet have any open access locations that we know about. I'd read "not_open_access" and immediately think `open_access_status = 'closed'`.

Maybe just "no_oa_locations" is a descriptive enough name for this scope without hinting at any extra meaning that isn't really there.

Naming aside, would we want to include any publications in this workflow based on the value of the `open_access_status` field? (This isn't a rhetorical question - I'm really asking.) For example, if there's a publication that has a known open access location, but we know that the open access location might only be temporary because we also know that the open_access_status is 'bronze', then would we still want to try deposit the file that we've been given?

Reply:
I'll change the name of that scope to be more specific.

For the second part, maybe.  I'm not too sure.  It may be something we want to confirm with Ana later, and consider adding in the future.